packages/cli/src/__tests__/daemon-config.test.ts

```diff
@@ -0,0 +1,52 @@
+import tmp from 'tmp-promise'
+import { writeFile } from 'node:fs/promises'
+import { DaemonConfig } from '../daemon-config'
+import { homedir } from 'node:os'
+
+describe('reading from file', () => {
+  let folder: tmp.DirectoryResult
+  let configFilepath: URL
+  beforeEach(async () => {
+    folder = await tmp.dir({ unsafeCleanup: true })
+    const base = new URL(`file://${folder.path}/`)
```

Comment:
I thought file: urls had three slashes ///

https://en.wikipedia.org/wiki/File_URI_scheme#How_many_slashes?

Reply:
It is three slashes indeed. Two are here in `file://` string, and another one is provided by `folder.path`, as it is an absolute path starting with `/`.